velox/connectors/hive/storage_adapters/s3fs/tests/S3FileTest.cpp

```diff
@@ -136,3 +137,24 @@ TEST_F(S3FileSystemTest, viaRegistry) {
   auto readFile = s3fs->openFileForRead(s3File);
   readData(readFile.get());
 }
+
+TEST_F(S3FileSystemTest, fileHandle) {
```

Comment:
I noticed a mis-match between the test name and file name: S3FileTest.cpp vs. S3FileSystemTest. Would you fix in a follow-up PR?

Reply:
Fixed here.